pages/users-page.ts

```diff
@@ -0,0 +1,34 @@
+// Users page before installation
+
+export class UsersPage {
+  readonly page: Page;
+  readonly defineUserButton: Locator;
+  readonly setRootPwdDots: Locator;
+  readonly setRootPwdMenuItem: Locator;
+  readonly backButton: Locator;
+  readonly noUserDefinedTxt: Locator;
+  readonly rootPwdNotSetTxt: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+    this.defineUserButton = page.getByRole('button', { name: 'Define a user now' });
+    this.setRootPwdDots =  page.getByRole('row', { name: 'Password Not set Actions' }).getByRole('button', { name: 'Actions' });
+    this.setRootPwdMenuItem =  page.getByRole('menuitem', { name: 'Set' });
+    this.backButton = page.getByRole('button', { name: 'Back' });
+    this.noUserDefinedTxt = page.getByText('No user defined yet');
+    this.rootPwdNotSetTxt = page.getByRole('row', { name: 'Password Not set Actions' }).getByRole('gridcell', { name: 'Not set' });
+  }
+  
+  async defineUser() {
+    await this.defineUserButton.click();
+  }
+  
+  async setRootPassword() {
+    await this.setRootPwdDots.click();
+    await this.setRootPwdMenuItem.click();
+  }
+  
+  async backToMainPage() {
```

Comment:
`back` should suffice, we might find ourselves in the future using this appended names in pages that do not come back where we expect because the screen flow change, we can make it simple from the beginning.

Reply:
ok